lib/plug/builder.ex

```diff
@@ -262,7 +262,7 @@ defmodule Plug.Builder do
 
       quote do
         require Logger
-        Logger.unquote(level)(unquote(message))
+        :ok = Logger.unquote(level)(unquote(message))
```

Comment:
I'd say also add a comment saying that the `_ =` is to please dialyzer; I'd be confused by reading it without any explanation :).